OpenRA.Game/Server/Server.cs

```diff
@@ -199,6 +239,15 @@ public Server(List<IPEndPoint> endpoints, ServerSettings settings, ModData modDa
 				}
 			};
 
+			if (Settings.RecordGameReplays)
+			{
+				recorder = new ReplayRecorder(() => { return Game.TimestampedFilename(extra: "-Server"); });
```

Comment:
Can you elaborate on the motivation for adding Server into the replay filenames?

Reply:
If there is a bug or something weird, we know where to look for.